.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -4,19 +4,23 @@ about: "If you're sure it's reproducible and not just your machine: submit an is
 
 ---
 
-# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+# Bug report
+
+**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**
```

Comment:
Can this PR be scoped down to just be these changes above (and same on feature suggestion)? If so: 👍🏻 to merge.